data/src/main/java/com/example/data/remote/api/FakeStoreApi.kt

```diff
@@ -2,11 +2,17 @@ package com.example.data.remote.api
 
 import com.example.domain.common.LoginCredentialsWrapper
 import com.example.domain.model.ApiResponse
+import com.example.domain.model.api_model.ApiUser
+import io.reactivex.rxjava3.core.Completable
 import io.reactivex.rxjava3.core.Single
 import retrofit2.http.Body
 import retrofit2.http.POST
+import retrofit2.http.PUT
 
 interface FakeStoreApi {
     @POST("/auth/login")
     fun loginUser(@Body loginCredentials: LoginCredentialsWrapper): Single<ApiResponse>
+
+    @PUT("/users/7")
+    fun updateUser(@Body user: ApiUser): Completable
```

Comment:
actually in the request '7' is an User id
But in our case (fake API) you can leave it as it is. Just make a comment for this request, describing why this is '7' and what should be if API, in example, become work normally